src/utils/mathUtils.ts

```diff
@@ -61,18 +61,11 @@ export module Utils {
     }
 
     /**
-     * Returns true **only** if x is NaN
-     */
-    export function isNaN(n: any) {
-      return n !== n;
-    }
-
-    /**
      * Returns true if the argument is a number, which is not NaN
      * Numbers represented as strings do not pass this function
      */
     export function isValidNumber(n: any) {
-      return typeof n === "number" && !Plottable.Utils.Math.isNaN(n) && isFinite(n);
+      return typeof n === "number" && n === n && isFinite(n);
```

Comment:
Leave isNaN as a private method. I don't like `n === n`. It is not expressive at all.